src/sentry/options/defaults.py

```diff
@@ -2918,5 +2918,10 @@
 register(
     "performance.event-tracker.sample-rate.transaction",
     default=0.0,
+)
+register(
+    "transactions.do_post_process_in_save",
+    type=Bool,
+    default=False,
```

Comment:
Instead of using a boolean, should this be a rollout rate so we can do a gradual deterministic rollout based on project or org ID? See for example

https://github.com/getsentry/sentry/blob/303b9302ede7598e84832e063c84790f3d579318/src/sentry/grouping/strategies/message.py#L58-L60